src/main/python/rlbot/utils/structures/game_interface.py

```diff
@@ -121,7 +121,8 @@ def start_match(self):
                                                 None if self.extension is None else self.extension.onMatchStart), None)
 
         if rlbot_status != 0:
-            raise rlbot_exception.RLBotException().raise_exception_from_error_code(rlbot_status)
+            exception_class = get_exception_from_error_code(rlbot_status)
+            raise exception_class()
```

Comment:
If you `raise` a class, it'll be instantiated implicitly. So we could do 

```raise get_exception_from_error_code(rlbot_status)```

and it'll have the same effect.

Reply:
That is actually really neat to know.